### PR TITLE
Prepare for release version 6.3.2

### PR DESCRIPTION
**Pull request by shervElmi**
### Changes proposed in this Pull Request:
With this Pull Request, we will release version 6.3.2

### Changelog:
* Update: The editor role can manage Business Directory Plugin.
* Update: Editor, Author, and Contributor roles can see a better design now.
* Fix: The editor role gets two links in the menu and one gives a 404 error.
* Fix: BD timing out due to images/icons.
* Fix: Displaying listing titles when a theme disables the single-page title fixed.
* Fix: The default theme styles aren't minified.
* Dev/update: All PHP codes are beautified.

### CODE REVIEW

File: README.TXT [hunk 1]
```diff
@@ -1,237 +1,246 @@
...
+
+= Is this plugin compatible with WordPress Multisite (WPMU)? =
+Yes it is. However, you cannot "network activate" the plugin (as this will share the database). Activate it on only the sub sites you need it.
+
+This can be done under Plugins -> Add New as the Administrator user. Do not "network activate" as the "super admin".
+
+== Changelog ==
+= 6.3.2 =
+* Update: Editor role can manage Business Directory Plugin.
+* Update: Editor, Author, and Contributor roles can see a better design now.
```
Comment (stephywells) [inline README.TXT:163]: Here are a few changes to help make the changes more clear. For fixes, I like to use past tense so it's clear the issue no longer exists.

* Update: Users with the Editor role can manage listings in the admin area.
* New: The default theme styles are now minified.
* Fix: Editors, Authors, and Contributors see the navigation sidebar now
* Fix: Editors were getting an extra link in the admin menu that led to a 404 page.
* Fix: On some sites, admin pages were timing out when loading the menu icons.
* Dev/update: All PHP codes are beautified

Can you tell me what this one means?
* Fix: Displaying listing titles when a theme disables the single-page title fixed.

Comment (shervElmi) [reply README.TXT:163]: The related issue is:
https://github.com/Strategy11/business-directory-premium/issues/44

File: README.TXT [hunk 2]
```diff
@@ -1,237 +1,246 @@
...
-* Fix: Some sites were getting an error during the database upgrade process.
-* Fix: Show the current variable plan price for logged out users.
-
-= 6.2.4 =
-* Fix: Redirect to the login page from the new listing form when logging in is required.
-* Fix: The listings weren't showing alphabetically in new installations, if the sorting settings hadn't been updated.
-
-<a href="https://businessdirectoryplugin.com/plugins/business-directory/?utm_source=wprepo&utm_medium=link&utm_campaign=liteversion">See changelog for all versions</a>
+=== Business Directory Plugin - Easy Listing Directories for WordPress ===
+Contributors: businessdirectoryplugin
```
Comment (stephywells) [inline README.TXT:2]: Would you mind changing this to:
Contributors: businessdirectoryplugin, sswells, srwells 

Then we can close my other open PR.

**Review by stephywells — CHANGES_REQUESTED** [review]
@shervElmi do you know why the whole readme is showing a diff?

File: README.TXT (hunk 1 above)
Comment (stephywells) [reply README.TXT:163]: So maybe this?
Fix: Listing titles were not always showing in the Enfold theme

**Comment by shervElmi** [comment]
> @shervElmi do you know why the whole readme is showing a diff?

Is it possible to change the Select end of Line Sequence?

File: README.TXT (hunk 2 above)
Comment (shervElmi) [reply README.TXT:2]: Of course!

**Comment by shervElmi** [comment]
> @shervElmi do you know why the whole readme is showing a diff?

@stephywells Let me revert this file and push the changes again.

**Comment by shervElmi** [comment]
> > @shervElmi do you know why the whole readme is showing a diff?
> 
> @stephywells Let me revert this file and push the changes again.

@stephywells I've reverted that on my local machine, and after that, I compared it with the new README.txt file, and Editor showed me only my changes!

I think it's ok, what do you think?

**Comment by shervElmi** [comment]
@stephywells I'm sure It's ok, and we can merge this PR.

**Review by stephywells — APPROVED** [review]
Looks good! I think the diff isn't a big deal on that file. You can go ahead and merge this.

**Comment by shervElmi** [comment]
> Looks good! I think the diff isn't a big deal on that file. You can go ahead and merge this.

Thank you for your support!

**Pull request merged**